src/main/java/org/opentripplanner/updater/alerts/GtfsEnhancedRealtimeAlertsUpdater.java

```diff
@@ -144,6 +157,10 @@ private FeedMessage parseJson(String data) throws Exception {
             alert.setEffect(GtfsRealtime.Alert.Effect.valueOf(alertNode.get("effect").textValue()));
         }
 
+        if (alertNode.get("cause") != null) {
```

Comment:
Do we need this?

Reply:
We don't specifically need this for that particular PR, but it's useful to see this field during debugging.